src/ansys/geometry/core/geometry/parameterization.py

```diff
@@ -202,6 +231,139 @@ def get_span(self) -> Real:
 
         return self.end - self.start
 
+    def get_relative_val(self, t: Real) -> Real:
+        """
+        Return an evaluation property of the interval, used in BoxUV.
```

Comment:
```suggestion
        Get an evaluation property of the interval, used in BoxUV.
```
I don't find this description very clear.